CHANGELOG.md

```diff
@@ -5,12 +5,20 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Updated dependencies for PureScript 0.14 (#16)
```

Comment:
I assume every pre-0.14 changelog update will need an entry like this.

Reply:
As per @hdgarrood I've replaced this line in all PRs with:

> Added support for PureScript 0.14 and dropped support for all previous versions